### PR TITLE
Check websocket readystate on destroy

### PR DESCRIPTION
**Pull request by groupboard**
Check websocket readyState before sending destroy message, otherwise it will generate a "InvalidStateError: The object is in an invalid state." exception if the websocket isn't in the open state.

### CODE REVIEW

File: html/janus.js [hunk 1]
```diff
@@ -1108,7 +1108,13 @@ function Janus(gatewayCallbacks) {
...
 			ws.addEventListener('error', onUnbindError);
 
-			ws.send(JSON.stringify(request));
+                        if (ws.readyState === 1) {
+                            ws.send(JSON.stringify(request));
+                        }
+                        else {
+                            onUnbindError();
+                        }
+
```
Comment (lminiero) [inline html/janus.js:1117]: Please use tabs, not spaces, for indentation. The `else` should be on the same line as the closing bracket too.

Comment (groupboard) [reply html/janus.js:1116]: Both fixed now.

Comment (lminiero) [reply html/janus.js:1116]: They don't seem fixed to me... I still see spaces used for indentation.

Comment (groupboard) [reply html/janus.js:1116]: Sorry, it was using 4-char indentation and I've changed it to 8 (so all tabs).

Comment (lminiero) [reply html/janus.js:1116]: It must not be char-indentation, only tab-indentation. I still see whitespaces.

Comment (lminiero) [reply html/janus.js:1116]: Edit: apologies, github was showing me an old thing... seems ok now! :+1:

**Comment by lminiero** [comment]
Thanks, merging!

**Pull request merged**